packages/loopring_v3/contracts/core/iface/IExchangeV3.sol

```diff
@@ -510,7 +509,6 @@ abstract contract IExchangeV3 is IExchange
         uint96  amount,
         address feeToken,
         uint96  fee,
-        uint    data,
```

Comment:
Brecht, will {from, token, storageID} uniquily identify a transfer? If so, maybe we can simply use these tree variables as the input to approveOffchainTransfer.

Reply:
It won't, because the operator could choose whatever the other transfer parameters are because `from` hasn't signed anything when the transfer is approved like this. So basically it has to contain pretty much all transfer data.

Reply:
So may be 'data' should be put inside "TransferAuxiliaryData" and be signed？

Reply:
Not sure I follow what you want to achieve here. Why is `data` necessary? And how can it be signed if it is approved using some onchain mechanism?